src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -197,6 +201,19 @@ function validate_boolean_string_value(value) {
     return false;
 }
 
+function validate_supplemental_groups(value) {
```

Comment:
please add type script annotations for new functions, it helps with finding bugs on js